plugins/modules/aci_bfd_interface_policy.py

```diff
@@ -0,0 +1,311 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Anvitha Jain (@anvjain)
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_bfd_interface_policy
```

Comment:
Should we combine this one with the multihop one? All the input is basically the same, only the classname differs.

Reply:
changed.

Reply:
how did you change this?

Reply:
guess this is decided not to be combined correct?

Reply:
Discussed with Lionel, and decided to keep this seperate since already built.